docs/tutorial/t-relate-kafka.md

```diff
@@ -0,0 +1,123 @@
+# Relate your Kafka deployment 
+
+This is part of the [Charmed Kafka Tutorial](/t/charmed-kafka-tutorial/). Please refer to this page for more information and the overview of the content. 
+
+## Relations
+Relations, or what Juju documentation [describes as Integration](https://juju.is/docs/sdk/integration), are the easiest way to create a user for Kafka in Charmed Kafka. Relations automatically create a username, password, and database for the desired user/application. As mentioned earlier in the [Access Kafka section](#access-kafka) it is a better practice to connect to Kafka via a specific user rather than the admin user.
+
+### Data Integrator Charm
+Before relating to a charmed application, we must first deploy our charmed application. In this tutorial we will relate to the [Data Integrator Charm](https://charmhub.io/data-integrator). This is a bare-bones charm that allows for central management of database users, providing support for different kinds of data platforms (e.g. MongoDB, MySQL, PostgreSQL, Kafka, OpenSearch, etc) with a consistent, opinionated and robust user experience. In order to deploy the Data Integrator Charm we can use the command `juju deploy` we have learned above:
```

Comment:
```suggestion
To start using the Kafka cluster, we will now relate our application to the [Data Integrator Charm](https://charmhub.io/data-integrator). This is a bare-bones charm that allows for central management of database users, providing support for different kinds of data platforms (e.g. MongoDB, MySQL, PostgreSQL, Kafka, OpenSearch, etc) with a consistent, opinionated and robust user experience. In order to deploy the Data Integrator Charm we can use the command `juju deploy` we have learned above:
```

Reply:
We are telling a story and the tutorial is supposed to be done from top to bottom. We don't have to tell the user that we must first kafka.